webpack/scenes/FlatpakRemotes/FlatpakRemotesPage.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { Table, Thead, Th, Tbody, Tr, Td } from '@patternfly/react-table';
+import TableIndexPage from 'foremanReact/components/PF4/TableIndexPage/TableIndexPage';
+import {
+  useSetParamsAndApiAndSearch,
+  useTableIndexAPIResponse,
+} from 'foremanReact/components/PF4/TableIndexPage/Table/TableIndexHooks';
+import { useTableSort } from 'foremanReact/components/PF4/Helpers/useTableSort';
+import EmptyPage from 'foremanReact/routes/common/EmptyPage';
+import Pagination from 'foremanReact/components/Pagination';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import { selectFlatpakRemotes, selectFlatpakRemotesError, selectFlatpakRemotesStatus } from './FlatpakRemotesSelectors';
+import { urlBuilder } from '../../__mocks__/foremanReact/common/urlHelpers';
+import { truncate } from '../../utils/helpers';
+
+const FlatpakRemotesPage = () => {
+  const response = useSelector(selectFlatpakRemotes);
+  const error = useSelector(selectFlatpakRemotesError);
+  const status = useSelector(selectFlatpakRemotesStatus);
+
+  const {
+    results = [], subtotal, page, per_page: perPage,
+  } = response || {};
+
+  const columnHeaders = [__('Name'), __('URL'), __('Last Scan')];
```

Comment:
```suggestion
  const columnHeaders = [__('Name'), __('URL'), __('Last scan')];
```

Except for page titles and main nav, we always use sentence case.

Reply:
I have updated it to sentence case